website/docs/configurations.md

```diff
@@ -15,6 +15,20 @@ This page covers the different ways of configuring your job to write/read Hudi t
 - [**Metrics Configs**](#METRICS): These set of configs are used to enable monitoring and reporting of keyHudi stats and metrics.
 - [**Record Payload Config**](#RECORD_PAYLOAD): This is the lowest level of customization offered by Hudi. Record payloads define how to produce new values to upsert based on incoming new record and stored old record. Hudi provides default implementations such as OverwriteWithLatestAvroPayload which simply update table with the latest/last-written record. This can be overridden to a custom class extending HoodieRecordPayload class, on both datasource and WriteClient levels.
 
+---
+Except directly passing configurations to Hudi jobs, since 0.10.0, Hudi also supports passing configurations through an external configuration file `hudi-default.conf` in which each line consists of a key and a value separated by whitespace/equal sign. For example:
+```
+hoodie.datasource.hive_sync.mode               jdbc
+hoodie.datasource.hive_sync.jdbcurl            jdbc:hive2://localhost:10000
+hoodie.datasource.hive_sync.support_timestamp  false
+```
+This is a cluster level configuration, all the Hudi jobs running in this cluster would share the same configuration.
+The configuration is parsed and evaluated when the Hudi engine processes are started. Changes to the configuration file require restarting the relevant processes.
+
```

Comment:
This statement is not true. As discussed it is not the engines like Spark, Hive that load the config. Instead we are relying on Hudi code paths to ultimately load this.

May be you should mention that this also kicks in via Spark SQL DML and that it helps reduces the configs that once would otherwise have to keep passing.

Reply:
Make sense. Removed